advanced_html_css/accessibility/wai_aria.md

```diff
@@ -58,29 +58,29 @@ When using such ARIA attributes, you would provide an `id` to one element, and y
 
 #### `aria-label`
 
-The `aria-label` attribute overrides any native label and modifies the name property in the accessibility tree, though it's best used when an element doesn't already have a native label. When you add `aria-label` to an element, you pass in a string as the value, which will become that elements accessible name. `aria-label` doesn't work on every HTML element, though. Adding the attribute to a plain `<div>` or a `<span>` will have no effect, for example.
+The `aria-label` attribute overrides any native label of an element and modifies its name property in the accessibility tree. It is best used when an element doesn't already have a native label. The string value of `aria-label` attribute  becomes the element's accessible name. `aria-label` does not have any effect on some HTML elements, such as `<div>` or a `<span>`. 
 
-A common use for this attribute can be for the "close" button often seen in menus or modals:
+A common use for `aria-label` can be found in the "close" buttons of menus or modals:
 
 ~~~html
 <button type='button' aria-label='Close menu'>X</button>
 ~~~
 
-Instead of a screen reader announcing, "X, button", which wouldn't make any sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our Semantic HTML lesson gets another shoutout... again!):
+Instead of a screen reader announcing, "X, button", which makes little sense to the user, it would announce, "Close menu, button". Another way you could use `aria-label` is on landmark elements (our Semantic HTML lesson gets another shoutout... again!):
 
 ~~~html
 <nav aria-label='main navigation'>...</nav>
 ~~~
 
 Once a screen reader reaches the above HTML, it would announce "Main navigation, navigation landmark". If you had multiple navigation elements on a page, you could give each a different `aria-label` value in order to separate them from one another, making them more understandable for screen reader users. Pretty neat, huh?
 
-One thing you should avoid using `aria-label` for is trying to change how a word is phonetically announced. Certain words may not get announced correctly by a screen reader, and you may have a temptation to try and fix this. To put it simply: **don't**. You may end up fixing how a word is announced by a screen reader, but that "fix" could end up making no sense when announced by other assistive technologies like a braille reader.
+Do not use `aria-label` to change how a word is phonetically announced. Certain words may not get announced correctly by a screen reader, and you may have a temptation to try and fix this. You may end up fixing how a word is announced by a screen reader, but that "fix" could end up making no sense when announced by other assistive technologies like a [braille](https://en.wikipedia.org/wiki/Braille) reader.
```

Comment:
```suggestion
**Do not** use `aria-label` to change how a word is phonetically announced. Certain words may not get announced correctly by a screen reader, and you may have a temptation to try and fix this. You may end up fixing how a word is announced by a screen reader, but that "fix" could end up making no sense when announced by other assistive technologies like a [braille](https://en.wikipedia.org/wiki/Braille) reader.
```
Putting more emphasis on this would be better since it is a common mistake.

Reply:
Agreed, thanks!